### PR TITLE
Created Sprint1.md

### LINKED ISSUE #35: Sprint1 Demo
## Description
As a project reviewer, I want to check the progress of the project at the end of the sprint.

I would like to see a demo of application and a documentation (Sprint1.md).

Sprint1.md should have points listing things delivered in planning, development, testing, and documentation phases of Sprints.

## Acceptance Criteria
- [x] Front End Video Demo 
- [x] Back End Video Demo
- [x] Add videos in repo
- [x] Sprint accomplishment documentation in Sprint1.md

### PR DESCRIPTION
**Pull request by Vineetkhadloya**
## Acceptance Criteria
- [x] Sprint accomplishment documentation in Sprint1.md

### CODE REVIEW

File: Sprint1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The goal for the Sprint 1 was to develop the essential Front-End Pages, API's and set-up the DB along with tables.
+
+Following REST API’s have been created :
+
+- Retrieve all Users : GET /api/users
+- Retrieve User based on User Id : GET /api/users/id/:id
+- Retrieve User based on Email Id : GET /api/users/email/:email
+- Add a New User : POST /api/users/newuser
+- Update User Details : POST /api/users/newuser
```
Comment (shashank136) [inline Sprint1.md]: It should be:

Update User Details: PUT /api/users/:id

I have corrected the error in API reference as well

File: Sprint1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Retrieve all Posts : GET /api/posts
+- Retrieve Post by Post Id : GET /api/posts/id/:id
+- Delete Post : DELETE /api/posts/:id
+
+Following Front-End Web Pages have been created :
+
+- Login Page
+- Registration Page 
+- Home Page 
+- Edit Page Page
```
Comment (shashank136) [inline Sprint1.md]: It should be Edit User page

**Review by bahetianjali — APPROVED** [review]

File: Sprint1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Following Tables have been created in DB :
```
Comment (shashank136) [inline Sprint1.md]: Are we going to mention tables in the database and schema?

**Review by bahetianjali — DISMISSED** [review]

**Review by shashank136 — DISMISSED** [review]
Approved.

**Review by bahetianjali — DISMISSED** [review]

**Review by haxxorsid — DISMISSED** [review]

**Review by shashank136 — DISMISSED** [review]
Approved

**Review by bahetianjali — DISMISSED** [review]

**Review by haxxorsid — APPROVED** [review]

**Review by Vineetkhadloya — COMMENTED** [review]
.

**Review by shashank136 — APPROVED** [review]
Approved

**Review by bahetianjali — APPROVED** [review]

**Pull request merged**